src/network/networkd-link.c

```diff
@@ -151,7 +151,7 @@ static bool link_ipv6_enabled(Link *link) {
         if (!socket_ipv6_is_supported())
                 return false;
 
-        if (link->network->bridge || link->network->bond)
+        if (link->network->bond)
```

Comment:
This is not right. assigning to bridge ipv6 addresses makes no sense.

Reply:
@ssahani and @tomty89 Yeah, It is strange to assigning addresses to slave, but it works :-)

Reply:
@yuwata is right. LL addressing is supported by kernel. Also it anonymous (bridge does not have any addresses) bridge slave have addresses.
```
14: veth0@veth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 0a:e8:65:c3:de:d8 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::8e8:65ff:fec3:ded8/64 scope link tentative 
       valid_lft forever preferred_lft forever
15: veth1@veth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br1 state UP group default qlen 1000
    link/ether f6:f0:cd:5c:01:74 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::f4f0:cdff:fe5c:174/64 scope link tentative 
       valid_lft forever preferred_lft forever
16: br1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether f6:f0:cd:5c:01:74 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::f4f0:cdff:fe5c:174/64 scope link tentative 
       valid_lft forever preferred_lft forever
```